src/transformers/generation_utils.py

```diff
@@ -660,6 +662,8 @@ def _get_logits_warper(
             warpers.append(TopPLogitsWarper(top_p=top_p, min_tokens_to_keep=(2 if num_beams > 1 else 1)))
         if typical_p is not None and typical_p < 1.0:
             warpers.append(TypicalLogitsWarper(mass=typical_p, min_tokens_to_keep=(2 if num_beams > 1 else 1)))
+        if renormalize_logits is True:
+            warpers.append(LogitNormalization())
```

Comment:
```suggestion
        # `LogitNormalization` should always be the last logit processor, when present
        if renormalize_logits is True:
            warpers.append(LogitNormalization())
```

Reply:
(that way we don't mistakenly add new processors after this one on this particular function, which actually is my number one concern :D )